draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -87,10 +87,10 @@ normative:
     target: https://www.tmforum.org/resources/suite/mtosi-4-0/
 
   IANA_YANG:
-    title: YANG Parameters
+    title: iana-hardware YANG Module
     author:
       org: IANA
-    target: https://www.iana.org/assignments/yang-parameters
+    target: https://www.iana.org/assignments/iana-hardware/
```

Comment:
Looking at draft-ietf-netmod-rfc8407bis, I am wondering whether it should be:

```suggestion
    target: https://www.iana.org/assignments/iana-hardware/iana-hardware.xhtml
```

Reply:
The OLD one is correct as redirection is in place to point to the more specific one you added